mscore/pianoroll.cpp

```diff
@@ -312,15 +309,15 @@ PianorollEditor::~PianorollEditor()
       }
 
 //---------------------------------------------------------
-//   focucOnElement
+//   focusOnElement
```

Comment:
Whatis the difference? I don't see any

Reply:
Fixed a typo in the comment (“focuc” to “focus”).

Reply:
Outsch, I've been looking really hard, but didn't spot it